src/sentry/static/sentry/app/views/settings/components/forms/projectMapperField.tsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {components} from 'react-select';
+
+import InputField from 'app/views/settings/components/forms/inputField';
+import {ProjectMapperType} from 'app/views/settings/components/forms/type';
+import SelectControl from 'app/components/forms/selectControl';
+import IdBadge from 'app/components/idBadge';
+import Button from 'app/components/button';
+import {IconAdd} from 'app/icons';
+import {t} from 'app/locale';
+
+type Props = InputField['props'];
+type RenderProps = Props & ProjectMapperType;
+
+export default class ProjectMapperField extends React.Component<Props> {
+  private sentryProjectRef = React.createRef<typeof SelectControl>();
+  private mappedRef = React.createRef<typeof SelectControl>();
```

Comment:
Do we use private fields?

Reply:
@EvanPurkhiser I guess we do because that's what the linter added lol. I didn't actually add them myself.

Reply:
I don't think our linter does this. Maybe VSCode did this for you?

Afaik we do not use private class fields atm

Reply:
@EvanPurkhiser yea idk actually..removed